tests/integration/cmd_devfile_init_test.go

```diff
@@ -159,11 +160,12 @@ var _ = Describe("odo devfile init command tests", func() {
 
 				for _, ctx := range []struct {
 					title, devfileVersion, requiredVersion string
+					gotLatest                              bool
```

Comment:
I think we can merge the changes from #6788 as well, use a dedicated `versionCheck` instead of `gotLatest`.